modules/pixi/PixiLayerRapidOverlay.js

```diff
@@ -0,0 +1,166 @@
+import * as PIXI from 'pixi.js';
+
+import { geojsonFeatures } from '../util/util.js';
+import { AbstractLayer } from './AbstractLayer.js';
+import { PixiFeatureLine } from './PixiFeatureLine.js';
+import { PixiFeaturePoint } from './PixiFeaturePoint.js';
+import { PixiFeaturePolygon } from './PixiFeaturePolygon.js';
+
+
+
+/**
+ * PixiLayerRapidOverlay
+ * This class contains any overlay vectors that should be 'drawn over' the map, usually at low zooms.
+ * The data for these are scraped from the RapidSystem's datasets, specifically the 'overlay' field.
+ * @class
+ */
+export class PixiLayerRapidOverlay extends AbstractLayer {
+
+  /**
+   * @constructor
+   * @param  scene    The Scene that owns this Layer
+   * @param  layerID  Unique string to use for the name of this Layer
+   */
+  constructor(scene, layerID) {
+    super(scene, layerID);
+    this._clear();
+    this._enabled = true;
+
+    const overlays = new PIXI.Container();
+    overlays.name = `${this.layerID}-overlays`;
+    overlays.sortableChildren = false;
+    overlays.interactiveChildren = true;
+    this.overlaysContainer = overlays;
+
+
+    const basemapContainer = this.scene.groups.get('basemap');
+    basemapContainer.addChild(overlays);
+  }
+
+
+  /**
+   * render
+   * Render the GeoJSON custom data
+   * @param  frame      Integer frame being rendered
+   * @param  viewport   Pixi viewport to use for rendering
+   * @param  zoom       Effective zoom to use for rendering
+   */
+  render(frame, viewport, zoom) {
+    if (!this.enabled || !(this.hasData())) return;
+
+    const vtService = this.context.services.vectortile;
+    const datasets = this.context.systems.rapid.datasets;
+
+    // for (const [key, dataset] in datasets) {
```

Comment:
Eventually this code should go through all the Rapid datasets and look for any overlays. For now it's hard-wired to just go looking for fbRoads. 

To keep the PR minimal, I am demonstrating footways overlays wired up to fbRoads, but that's just for demo purposes. Footways impl is in a different branch.

Reply:
I have since rewired the PR to look through all the datasets, examine which ones have overlays, and conditionally display those overlays IF the dataset is enabled.